server/rpm/pbench-server.spec.j2

```diff
@@ -116,6 +116,16 @@ fi
 /%{installdir}/lib/pbench/indexer.py
 /%{installdir}/lib/pbench/mock.py
 /%{installdir}/lib/pbench/report.py
+/%{installdir}/lib/pbench/api/__init__.py
+/%{installdir}/lib/pbench/api/process_tb.py
+/%{installdir}/lib/pbench/common.py
+/%{installdir}/lib/pbench/test/__init__.py
+/%{installdir}/lib/pbench/test/conftest.py
+/%{installdir}/lib/pbench/test/fixtures/upload/bad_tarball/bad_log.tar.xz.md5
+/%{installdir}/lib/pbench/test/test-fixtures/upload/log.tar.xz
+/%{installdir}/lib/pbench/test/test-fixtures/upload/log.tar.xz.md5
+/%{installdir}/lib/pbench/test/unit/__init__.py
+/%{installdir}/lib/pbench/test/unit/test_process_tb.py
```

Comment:
We don't want to *install* the unit test files; they're just in the development tree.

Reply:
yeah, I didn't want to include them in the commit. While I was building RPM, rpm was expecting these files as well else it was throwing error So, I included them.